packages/edit-post/src/components/keyboard-shortcut-help-modal/shortcut.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+const mapKeyCombination = ( keyCombination ) => keyCombination.map( ( character, index ) => {
+	if ( character === '+' ) {
+		return (
+			<Fragment key={ index }>
+				{ character }
+			</Fragment>
+		);
+	}
+
+	return (
+		<kbd
+			key={ index }
+			className="edit-post-keyboard-shortcut-help__shortcut-key"
+		>
+			{ character }
+		</kbd>
+	);
+} );
+
+function Shortcut( { description, keyCombination, ariaLabel } ) {
+	return (
+		<>
+			<div className="edit-post-keyboard-shortcut-help__shortcut-description">
```

Comment:
This should be:

```suggestion
			<div className="edit-post-keyboard-shortcut-help-modal__shortcut-description">
```

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

Same applies to other classes in the file. And in the other file as well.

Reply:
These are not new classes but I'll fix them.